.github/scripts/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+fpga_fam=$1
+shift
+
+# select particular example for xc7
+examples="$@"
+if [ "$fpga_fam" == "xc7" -a -z "$examples" ] ; then
+    examples="counter picosoc linux_litex"
+fi;
+
+# activate conda and enter example dir
+eval $(tuttest docs/building-examples.rst export-install-dir 2>&1)
+eval $(tuttest docs/building-examples.rst fpga-fam-$fpga_fam 2>&1)
+eval $(tuttest docs/building-examples.rst conda-prep-env 2>&1)
+eval $(tuttest docs/building-examples.rst conda-act-env 2>&1)
+eval $(tuttest docs/building-examples.rst enter-dir-$fpga_fam 2>&1)
+
+if [ "$fpga_fam" = "xc7" ]; then
+    # Xilinx 7-Series examples
```

Comment:
It seems like which examples are supported by which boards should be part of the documentation? Can we maybe use something like "selectors" with tuttest in some way?

Reply:
We added wildcards support to tuttest:
https://github.com/antmicro/tuttest/pull/26